composer.json

```diff
@@ -61,5 +62,6 @@
         "branch-alias": {
             "dev-master": "1.9.x-dev"
         }
-    }
+    },
+    "minimum-stability": "dev"
```

Comment:
Should that really be added?

Reply:
That would be required to get the beta2 until released, but I don't think it'll stay here. That was just the way to install it locally for me.
Anyway, I'm waiting some instructions/plans/some help to deal with the CI as we're even unlikely to keep the messenger component in `require-dev` due to the sf 2.7 support in this bundle.